apps/wa/wa-task-batch-reconfiguration/demo-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-reconfiguration
+spec:
+  values:
+    job:
```

Comment:
missing kustomizes for all envs except for AAT?

Reply:
Added missing env